index.html

```diff
@@ -1,11 +1,10 @@
-!DOCTYPE html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="styles.css">
     <title>Microverse Product Landing Page</title>
   </head>
-     <header id="header">

```

Comment:
Good work on your first HTML project. You may please check your page structure and follow the correct HTML page flow.  You may please remove the extra header element left on the page